packages/hardhat-core/src/internal/core/config/default-config.ts

```diff
@@ -141,6 +141,30 @@ export const defaultHardhatNetworkParams: Omit<
         ]),
       },
     ],
+    [
```

Comment:
This is ok, but i'd rather keep track of it, either with a comment or an issue (linked in a comment).

Reply:
Done in https://github.com/NomicFoundation/hardhat/pull/5394/commits/1ad1973595c53e57cffd7570f0444d68b6c36b80